composer.json

```diff
@@ -12,7 +12,7 @@
         "drupal/entity_reference_revisions": "^1.10",
         "drupal/link_description": "^1.0",
         "drupal/time_field": "^2.1",
-        "openeuropa/oe_content": "^3.1"
+        "openeuropa/oe_content": "^4.1"
```

Comment:
we can't drop version 3, this is open source, we dont know if anybody else is using it so we have to support both major versions